homeassistant/components/unifi/sensor.py

```diff
@@ -192,6 +199,88 @@ def async_device_state_value_fn(hub: UnifiHub, device: Device) -> str:
     return DEVICE_STATES[device.state]
 
 
+@callback
+def async_device_wan_monitor_latency_supported_fn(
+    wan: Literal["WAN", "WAN2"],
+    monitor_target: str,
+    hub: UnifiHub,
+    obj_id: str,
+) -> bool:
+    """Determine if an device have a latency monitor."""
+
+    device = hub.api.devices[obj_id]
+
+    if device is None or device.uptime_stats is None:
```

Comment:
device can not be None as devices[obj_id] expects obj_id to exist

```suggestion
    if (device := hub.api.devices[obj_id]) and device.uptime_stats is None:
```
something like this